libraries/Microsoft.Bot.Schema/EventActivityNames.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Schema
+{
+    /// <summary>
+    /// Define values for common event names used by activities of type <see cref="ActivityTypes.Event"/>.
+    /// </summary>
+    public static class EventActivityNames
```

Comment:
It seems like the naming should be `ActivityEventNames` to be consistent with `ActivityTypes`.

Reply:
Renamed, thanks for the suggestion!